tests/tensorflow_autolog/test_tensorflow2_autolog.py

```diff
@@ -31,33 +31,78 @@ def random_train_data():
 
 
 @pytest.fixture
-def tf_keras_random_data_run(random_train_data):
-    mlflow.tensorflow.autolog(every_n_iter=5)
+def random_one_hot_labels():
+    n, n_class = (1000, 10)
+    classes = np.random.randint(0, n_class, n)
+    labels = np.zeros((n, n_class))
+    labels[np.arange(n), classes] = 1
+    return labels
+
+
+@pytest.fixture(params=[True, False])
+def manual_run(request):
+    return request.param
```

Comment:
Actually, I wonder if we could actually make this fixture start & end the run so that we don't need to do so in each of our tests, e.g.:


```suggestion
    if request.param:
        mlflow.start_run()
    yield
    mlflow.end_run()
```

The `yield` statement will pause execution of the fixture till after the test completes, and then `mlflow.end_run()` will be called after the test completes